src/polyfills/domtokenlist-toggle.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isIe} from '../service/platform-impl';
+
+/**
+ * Polyfill for `DOMTokenList.prototype.toggle(token, opt_force)` method.
+ * This is specially important because IE does not support `opt_force` attribute.
+ * See https://goo.gl/hgKNYY for details.
+ * @param {string} token
+ * @param {boolean=} opt_force
+ * @this {DOMTokenList}
+ * @return {boolean}
+ */
+function domTokenListTogglePolyfill(token, opt_force) {
+  const remove = opt_force === undefined ? this.contains(token) : !opt_force;
+  if (remove) {
+    this.remove(token);
+    return false;
+  } else {
+    this.add(token);
+    return true;
+  }
+}
+
+
+/**
+ * Polyfills `DOMTokenList.prototype.toggle` API in IE.
+ * @param {!Window} win
+ */
+export function install(win) {
+  const ua = win.navigator.userAgent;
+  if (isIe(ua) && win.DOMTokenList && win.DOMTokenList.prototype.toggle) {
```

Comment:
https://github.com/ampproject/amphtml/pull/6400#discussion_r90308940. Shouldn't we always be polyfilling?